File: src/herbmixing.py
g, r = [int(d) for d in input().split()]
ans = 0
if g > 0:
    ans += 10 * min(g, r)
    remain = max(g - r, 0)
    ans += 10 * (remain // 3)
    remain %= 3
    ans += 3 * (remain // 2)
    remain %= 2
    ans += remain
print(ans)

File: src/herbmixing.lua
g, r = io.read("*n", "*n")
ans = 0
if g > 0 then
    ans = ans + 10 * math.min(g, r)
    remain = math.max(g - r, 0)
    ans = ans + 10 * (remain // 3)
    remain = remain % 3
    ans = ans + 3 * (remain // 2)
    remain = remain % 2
    ans = ans + remain
end
print(ans)
